### PR TITLE
use GitHub actions

### PR DESCRIPTION
**Pull request by jopecko**
https://github.com/typelevel/discipline-scalatest/issues/132

### CODE REVIEW

**Comment by jopecko** [comment]
for your consideration, @larsrh and @djspiewak

**Review by djspiewak — CHANGES_REQUESTED** [review]
I can re-push this to upstream so that the actions are picked up and run! Thank you for doing this!

File: build.sbt [hunk 1]
```diff
@@ -1,4 +1,31 @@
-import sbtcrossproject.CrossPlugin.autoImport.{crossProject, CrossType}
+val Scala212 = "2.12.11"
+
+ThisBuild / crossScalaVersions := Seq("2.13.3", scalaVersion.value, "2.11.12", "3.0.0-M1")
```
[the remark this answers is not in the document]
Comment (jopecko) [reply build.sbt]: Will do. Thanks for the sbt plugin that makes this so easy to do!

**Comment by larsrh** [comment]
Thanks a lot!

**Pull request closed without merging**